doc/md/timers.md

```diff
@@ -0,0 +1,9 @@
+# Timers
+
+Internet Computer canisters can set an arbitrary number of single-expiration or recurring timers. See the `Timers.mo` module in the base library.
+
+The underlying mechanism is a [canister global timer](https://internetcomputer.org/docs/current/references/ic-interface-spec#timer) that is reprogrammed according to an internally kept priority queue.
+
+The timer mechanism can be disabled completely by passing the `-no-timer` flag to `moc`.
```

Comment:
Is the flag documented in compiler.md (or whatever it's called)?

Reply:
Adding...